test/spec/change-streams/unified/change-streams.json

```diff
@@ -1,10 +1,21 @@
 {
   "description": "change-streams",
   "schemaVersion": "1.0",
+  "runOnRequirements": [
```

Comment:
@nbbeeken Please double check to make sure all the tests in the spec directory that have been added match to the expected spec updates

Reply:
Confirmed, everything is synced and up to date. I caught the listCollection case that we may want to consider testing.

Reply:
Let's leave this comment open to track that additional test

Reply:
I added tests for listCollection, listDatabases and listIndexes!

Reply:
Test LGTM, some organizational requests discussed on slack

Reply:
Done!